terraform/modules/pubsub/pubsub.tf

```diff
@@ -98,5 +99,6 @@ output "queue" {
     topic             = google_pubsub_topic.task.name
     subscription_kind = "gcp-pubsub"
     subscription      = google_pubsub_subscription.task.name
+    dead_letter_topic = google_pubsub_topic.dead_letter.name
```

Comment:
IIUC, this means we will publish unretryable errors to the existing PubSub dead letter topic (or dead letter SNS topic on the AWS side). I don't think this is what we want, because those dead letters will trigger our existing alerting rules for dead letters. I think dead letters from unretryable failures like packet decryption failure should go to a different dead letter topic and queue so that we can be aware of them but not alert.

Reply:
I'll work on the new topic/queue next. How about "...-rejected" as a name?

Reply:
That sounds good. I think we should also put some alerts on these new queues, but we can discuss that separately from this PR.

Reply:
+1 to putting this in another PR, but I'd ideally like to see alerts before we release this to production: I totally agree that dropping the occasional packet decryption error is fine, but if we see a large volume of these errors I'd like to know (since this might effectively mean we are currently dropping data on the floor in an unrecoverable way).

Reply:
I thought about this a little more; I'm still worried about the risk of us forgetting that `is_retryable = false` also means that we don't alert on the failed work item (unless it is high volume). But this is fairly hypothetical--I definitely don't want to complicate an already-large PR with this.

Would you mind adding to the `is_retryable` doc string that this also effectively also changes DLQ routing/alerting? This is sort of a "layering violation" but I think it's important enough that it should be documented, and it may not be immediately obvious to someone who is "just adding a new error case" in the future.